WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/WooAboutScreenConfiguration.swift

```diff
@@ -0,0 +1,210 @@
+                }),
+                AboutItem(title: Titles.website, subtitle: Subtitles.website, cellStyle: .value1, action: { [weak self] context in
+                    self?.present(url: Links.website, from: context.viewController)
+                }),
+                AboutItem(title: Titles.blog, subtitle: Subtitles.blog, cellStyle: .value1, action: { [weak self] context in
+                    self?.present(url: Links.blog, from: context.viewController)
+                })
+            ],
+            [
+                AboutItem(title: Titles.legalAndMore, accessoryType: .disclosureIndicator, action: { context in
+                    context.showSubmenu(title: Titles.legalAndMore, configuration: WooLegalAndMoreSubmenuConfiguration())
+                })
+            ],
+            [
+                AboutItem(title: Titles.automatticFamily, accessoryType: .disclosureIndicator, hidesSeparator: true, action: { [weak self] context in
+                    self?.present(url: Links.automatticFamily, from: context.viewController)
+                }),
+                AboutItem(title: "", cellStyle: .appLogos)
+            ],
+            [
+                AboutItem(title: Titles.workWithUs,
+                          subtitle: Subtitles.workWithUs,
+                          cellStyle: .subtitle,
+                          accessoryType: .disclosureIndicator,
+                          action: { [weak self] context in
+                              self?.present(url: Links.workWithUs, from: context.viewController)
+                          })
+            ]
+        ]
+    }
+
+// MARK: - Presentation Actions
+
+    private func present(url: URL, from viewController: UIViewController) {
+        let vc = SFSafariViewController(url: url)
+        viewController.present(vc, animated: true, completion: nil)
+    }
+
+    private func presentShareSheet(from viewController: UIViewController, sourceView: UIView?) {
+        let activityItems = [
+            ShareAppTextActivityItemSource(message: TextContent.shareSheetMessage) as Any,
+            Links.website as Any
+        ]
+
+        let activityViewController = UIActivityViewController(activityItems: activityItems, applicationActivities: nil)
+        activityViewController.popoverPresentationController?.sourceView = sourceView
+        viewController.present(activityViewController, animated: true, completion: nil)
+    }
+
+    func dismissScreen(_ actionContext: AboutItemActionContext) {
+        actionContext.viewController.presentingViewController?.dismiss(animated: true)
+    }
+
+    func willShow(viewController: UIViewController) {}
+
+    func didHide(viewController: UIViewController) {}
+
+// MARK: - Header Info
+
+    // Provides app info to display in the header of the about screen.
+    //
+    static var appInfo: AboutScreenAppInfo {
+        let name = (Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as? String) ?? ""
```

Comment:
Can you override this value and call the app WooCommerce? The shortened name of "Woo" feels weird when I see it on the screen. 😉

Reply:
Sure!